### ISSUE TITLE
Build fails when linking manually or automatically on iOS. 

### ISSUE THREAD

**Issue by Stophface** [zoontek/react-native-permissions#148]
I experience an error on iOS when I try to use the library.

I tried to link the library automatically with `react-native link react-native-permissions`. Doing it that way, throws me the error `undefined is not an object (evaluating 'RNPermissions.getPermissionStatus')`

I tried to link the library manually: 
I put the file `libReactNativePermissions.a` which I found in `Libraries - ReactNativePermissions.xcodeproj - ReactNativePermissions - Products - libReactNativePermissions.a` through Drag & Drop into `Build Phases - Link Binary with Libraries` of my root xCode project. When I run `react-native run-ios` I get:

```
** BUILD FAILED **

The following build commands failed:

	CompileC /Users/.../myApp/_app/myApp/ios/build/Build/Intermediates/ReactNativePermissions.build/Debug-iphonesimulator/ReactNativePermissions.build/Objects-normal/x86_64/ReactNativePermissions.o ReactNativePermissions.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)

Installing build/Build/Products/Debug-iphonesimulator/myApp.app
Launching org.reactjs.native.example.myApp
```

Interestingly, the iPhone simulator still starts. When I go to the place where the `Permission Library` is invoked, I get the same error as described above. 

I installed `react-native-permission` library with `yarn`. It works fine on `Android`.
react-native-cli: 2.0.1
react-native: 0.48.3

**Comment by yonahforst** [comment]
Hi @Stophface - it definitely sounds like a linking problem. Have you tried removing the library, unlinking it, and doing it all over again? I'm not sure what the steps are for unlinking, but I'm sure you can find something online

**Comment by Stophface** [comment]
@yonahforst Yes I tried that. 
No luck. I also uninstalled `react-native` and all dependencies in all folders as well as`global node moduls` and reinstalled a clean project with only the `Permissions` library. It works on `android`, but it cannot compile on `iOS`. It points me to a `.m` file where `RCT bridge` is identified double or something... That file is only highlighted when you try to `build` the project in `xCode`though. The error messages are more detailed there. Maybe the library is broken with the newer `react-native` version?

**Comment by yonahforst** [comment]
@Stophface - could be. Which version of the library are using? I'm merging some fixes now and will be releasing a new version shortly

**Comment by Stophface** [comment]
@yonahforst the latest. rn and rn permissions

**Comment by yonahforst** [comment]
you can try building from master? or wait until I release a new version

**Comment by aaronbuchanan** [comment]
getting the same issue here, I'll try to build from master

**Comment by aaronbuchanan** [comment]
```** BUILD SUCCEEDED **```

@yonahforst any sense of when you'll drop a release to npm? Thanks for your work on this lib!!

**Comment by yonahforst** [comment]
published 1.0.1 🎉
